### PR TITLE
Updating ach example

### PR DESCRIPTION
**Pull request by abhipillai**
Please explain the changes you made here.
Updates ACH example and removes the card element.

_Does this close any currently open issues?_

- [x] [Individual Contributor License Agreement (CLA)](https://spreadsheets.google.com/spreadsheet/viewform?formkey=dDViT2xzUHAwRkI3X3k5Z0lQM091OGc6MQ&ndplr=1) signed

### CODE REVIEW

File: public/examples/ach.html [hunk 1]
```diff
@@ -113,14 +122,6 @@
...
         }
 
-        let card;
-        try {
-          card = await initializeCard(payments);
-        } catch (e) {
-          console.error('Initializing Card failed', e);
-          return;
-        }
-
```
Comment (nwilmessquare) [inline public/examples/ach.html:123]: Thanks for removing the card flow from this example.  It just muddles up the example as is.

File: public/examples/ach.html [hunk 2]
```diff
@@ -137,33 +138,24 @@
...
           try {
-            // disable the submit button as we await tokenization and make a payment request.
-            cardButton.disabled = true;
             achButton.disabled = true;
-            const token = await tokenize(paymentMethod, options);
-            const paymentResults = await createPayment(token);
-            displayPaymentResults('SUCCESS');
-
-            console.debug('Payment Success', paymentResults);
+            tokenize(paymentMethod, options);
```
Comment (nwilmessquare) [inline public/examples/ach.html:142]: I know ACH relies mostly on the event listener, but I feel like this should still be awaited..?

File: public/examples/ach.html [hunk 3]
```diff
@@ -137,33 +138,24 @@
...
-          await handlePaymentMethodSubmission(event, card);
-        });
-
-        const achButton = document.getElementById('ach-button');
-        achButton.addEventListener('click', async function (event) {
-          const paymentForm = document.getElementById('payment-form');
-          const achOptions = getACHOptions(paymentForm);
-          await handlePaymentMethodSubmission(event, ach, achOptions);
-        });
+        // Checkpoint 2.
```
Comment (nwilmessquare) [inline public/examples/ach.html:150]: This checkpoint is directly at odds with the idea of this example being completely focused on ACH and ACH alone, I think checkpoint 1 was the card example.  We should pick a lane here - each payment method is independent, or the examples build on each other.

File: public/examples/ach.html (hunk 2 above)
Comment (abhipillai) [reply public/examples/ach.html:142]: All the errors will now be captured via the listener so we don't need to await this anymore.

File: public/examples/ach.html (hunk 3 above)
Comment (abhipillai) [reply public/examples/ach.html:150]: good callout

File: public/examples/ach.html (hunk 2 above)
Comment (nwilmessquare) [reply public/examples/ach.html:142]: In that case, the async keyword should be removed from the tokenize() method definition on line 50.

**Pull request merged**